schema/examples/direct/multiple_signers/manifest.xml

```diff
@@ -1,10 +1,18 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <direct-signature-job-manifest xmlns="http://signering.posten.no/schema/v1">
     <signer>
-        <personal-identification-number>12345678910</personal-identification-number>
+        <reference>123abc</reference>
+        <signing-ceremony>
+            <method>ELECTRONIC_SIGNATURE</method>
+            <authentication-level>3</authentication-level>
```

Comment:
Er dette sensitivitetsnivået til dokumentet, eller den påkrevde for autentiseringsbasert signering?